vividus-to-xray-exporter/src/main/java/org/vividus/xray/facade/XrayFacade.java

```diff
@@ -75,6 +79,17 @@ public void updateTestCase(String testCaseKey, TestCaseParameters testCaseParame
         LOGGER.atInfo().addArgument(testCaseKey).log("Test with key {} has been updated");
     }
 
+    public void updateTestExecution(String testExecutionKey, List<String> testCaseKeys) throws IOException
+    {
+        TestExecution testExecution = new TestExecution(testCaseKeys);
+        String testExecutionRequest = objectMapper.writeValueAsString(testExecution);
+        LOGGER.atInfo()
+              .addArgument(() -> StringUtils.join(testCaseKeys, ", "))
+              .addArgument(testExecutionKey)
+              .log("Add {} test cases to {} test execution");
```

Comment:
`Link` or `Add` ?

Reply:
from API point of view add, I'll ask end users

Reply:
decided to use Add